integration/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.8'
+services:
+  squid:
+    image: ubuntu/squid:latest
```

Comment:
This is just so people don't have to install `squid` locally to run integration tests.